test/metadata_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+require 'ttnt/metadata'
+
+class MetaDataTest < TTNT::TestCase
+  def setup
+    @storage_file = "#{@repo.workdir}/.ttnt"
+    FileUtils.rm_rf(@storage_file)
```

Comment:
You could simply use `File.delete` here ; this seems less overkill and more safe anyway.